### PR TITLE
Add Contao 4 TinyMCE customisation instructions.

### PR DESCRIPTION
**Pull request by christianromeni**
This adds a how to change TinyMCE in Contao 4

### CODE REVIEW

**Comment by christianromeni** [comment]
I was unsure what to do with the **Tag** Part, so I just copied it.. Someone please change it the way its supposed to be.

File: cookbook/customizing-contao/customizing-tinymce.md [hunk 1]
```diff
@@ -21,6 +21,46 @@ $GLOBALS['TL_DCA']['tl_content']['fields']['text']['eval']['rte'] =
 'tinyCustom';
 ```
 
+**Tag** The text below concerns the version 4.0.0 and higher.
 
-[1]: http://tinymce.moxiecode.com
-[2]: custom-configurations.md#customizing-the-data-container-configuration
+![](images/tinyMCE4.jpg)
+
+The image shows the default configuration of the editor in the Contao default be_tinyMCE.html5 Template.
```
Comment (lionel-m) [inline cookbook/customizing-contao/customizing-tinymce.md]: Could you add the backtick quotes (`) for file name and path please.

`be_tinyMCE.html5`, `/template/mytemplates/`

File: cookbook/customizing-contao/customizing-tinymce.md [hunk 2]
```diff
@@ -21,6 +21,46 @@ $GLOBALS['TL_DCA']['tl_content']['fields']['text']['eval']['rte'] =
...
+```js
+// Define a color:
+  style_formats_merge: true,
+  style_formats: [
+    { title: 'Text Red', inline: 'span', styles: { 'color': '#d11216' }, classes: 'text-red' },
+  ]
+```
+
+Important is the line `style_formats_merge: true`.
+This allows adding own formates without overwrighting the default formats that already exist.
```
Comment (lionel-m) [inline cookbook/customizing-contao/customizing-tinymce.md]: This allows adding your (or our) own formats without overwriting ...

File: cookbook/customizing-contao/customizing-tinymce.md [hunk 3]
```diff
@@ -21,6 +21,46 @@ $GLOBALS['TL_DCA']['tl_content']['fields']['text']['eval']['rte'] =
...
+
+![](images/tinyMCE4Custom.jpg)
+
+To clean up Custom formats, you can group them inside your own format folder like this:
```
Comment (lionel-m) [inline cookbook/customizing-contao/customizing-tinymce.md]: Custom -> custom

File: cookbook/customizing-contao/customizing-tinymce.md (hunk 2 above)
Comment (christianromeni) [inline cookbook/customizing-contao/customizing-tinymce.md]: Done

File: cookbook/customizing-contao/customizing-tinymce.md (hunk 3 above)
Comment (christianromeni) [inline cookbook/customizing-contao/customizing-tinymce.md]: Done

File: cookbook/customizing-contao/customizing-tinymce.md [hunk 4]
```diff
@@ -41,11 +41,11 @@ This is how you can add custom formats to TinyMCE:
 ```
 
 Important is the line `style_formats_merge: true`.
-This allows adding own formates without overwrighting the default formats that already exist.
+This allows adding your formates without overwrighting the default formats that already exist.
```
Comment (lionel-m) [inline cookbook/customizing-contao/customizing-tinymce.md]: **formats** without **overwriting**

Comment (christianromeni) [inline cookbook/customizing-contao/customizing-tinymce.md]: Done

Comment (lionel-m) [inline cookbook/customizing-contao/customizing-tinymce.md]: Thanks 👍

File: cookbook/customizing-contao/customizing-tinymce.md [hunk 5]
```diff
@@ -21,6 +21,46 @@ $GLOBALS['TL_DCA']['tl_content']['fields']['text']['eval']['rte'] =
 'tinyCustom';
 ```
 
+**Tag** The text below concerns the version 4.0.0 and higher.
 
-[1]: http://tinymce.moxiecode.com
-[2]: custom-configurations.md#customizing-the-data-container-configuration
+![](images/tinyMCE4.jpg)
+
+The image shows the default configuration of the editor in the Contao default `be_tinyMCE.html5` Template.
```
Comment (lionel-m) [inline cookbook/customizing-contao/customizing-tinymce.md]: Template -> template

File: cookbook/customizing-contao/customizing-tinymce.md [hunk 6]
```diff
@@ -21,6 +21,46 @@ $GLOBALS['TL_DCA']['tl_content']['fields']['text']['eval']['rte'] =
 'tinyCustom';
 ```
 
+**Tag** The text below concerns the version 4.0.0 and higher.
```
Comment (lionel-m) [inline cookbook/customizing-contao/customizing-tinymce.md]: The symbol `>` should be added before `**Tag**`

`> **Tag** The text ...`

Comment (christianromeni) [inline cookbook/customizing-contao/customizing-tinymce.md]: I'm realy not sure what to write here..?

Comment (lionel-m) [inline cookbook/customizing-contao/customizing-tinymce.md]: @christianromeni The GitBook plugin for creating these kinds of blocks was replaced by a new one. @aschempp Should we write like this? 

For example:

```
> #### note:: Compatibility note 
> The text below concerns the version 4.0.0 and higher.
```

**Comment by aschempp** [comment]
If I understand correctly, the adjustments to add custom styling work pretty much the same in Contao 3.5 and Contao 4, is that right?

I suggest we also update the general information about TinyMCE, would you take care of that or should I do that when merging the changes?

**Comment by christianromeni** [comment]
Yes, the custom styling part works the same.. Only where the changes should be done is different.

It would be nice if you could do it when merging. Thanks ;)

**Pull request merged**